src/Components/Filters/Search.jsx

```diff
@@ -1,53 +1,28 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
 import Styles from './Search.styles.scss';
-import Dropdown from './../Common/Dropdown';
+import { useNetwork } from '../../state/network/Context';
 
-const SEARCH_CATEGORY = ['url', 'body'];
+const Search = () => {
+  const { state, actions } = useNetwork();
```

Comment:
This seems like every time nay part of the state object changes this component will re-render. Not sure how big is the effort to make it subscribed to search value changes

Reply:
I changed it back to pass in the values until I refactor the reducer.